controllers/loginController.js

```diff
@@ -1,73 +1,75 @@
-var bodyParser=require('body-parser');
-module.exports=function(app){
-/* MONGOOSE SETUP */
-const mongoose = require('mongoose');
-mongoose.connect('mongodb://username:password@databaseserver:port/databasename',{ useNewUrlParser: true});
-/*Replace the above connection string with the actual connection string of your MongoDB database*/
-const Schema = mongoose.Schema;
-const UserDetail = new Schema({
-      username: String,
-      password: String
+const bodyParser = require('body-parser');
+const dotenv = require('dotenv');
+dotenv.config();
+module.exports = function (app) {
+    /* MONGOOSE SETUP */
+    const mongoose = require('mongoose');

```

Comment:
you should have database connection only in one place (you can place it in app.js instead of the 2 controllers)